ra2ce/analyses/indirect/analyses_indirect.py

```diff
@@ -349,43 +361,56 @@ def multi_link_redundancy(
 
             for edges in edges_remove:
                 u, v, k, edata = edges
-
+                time = round((edata["length"] * 1e-3) / edata["avgspeed"], 2)  # in hours and avg speed in km/h
                 if nx.has_path(graph, u, v):
-                    alt_dist = nx.dijkstra_path_length(
-                        graph, u, v, weight=analysis.weighing.config_value
-                    )
+                    alt_dist = nx.dijkstra_path_length(graph, u, v, weight="length")
                     alt_nodes = nx.dijkstra_path(graph, u, v)
                     connected = 1
+                    if analysis.weighing.config_value == "time":
```

Comment:
You should not be comparing to a string but rather to an `enum`, the purpose of `config_value` its simply to normalize the output when logging / printing / exporting.
```suggestion
                    if analysis.weighing == WeighingEnum.TIME:
```